lib/screens/forgot_password.dart

```diff
@@ -122,15 +77,13 @@ class _ForgotPasswordState extends State<ForgotPassword> {
                         height: 15,
                       ),
                       ElevatedButton(
-                        onPressed: conditionToShowAlertDialog,
-                        style: Theme.of(context)
-                            .elevatedButtonTheme
-                            .style!
-                            .copyWith(
-                              backgroundColor: MaterialStateProperty.all<Color>(
-                                CommonColors.lightGreenColor,
-                              ),
-                            ),
+                        onPressed:
+                            readForgotPasswordStore.conditionToShowAlertDialog,
+                        style: theme.elevatedButtonTheme.style!.copyWith(
```

Comment:
Never force unwrap in styling always use the null safe option like **theme.elevatedButtonTheme.style?.copyWith**